frontend/coprs_frontend/coprs/templates/coprs/detail/package.html

```diff
@@ -56,6 +56,11 @@ <h3 class="panel-title">General Information</h3>
           <dd>{{ package.max_builds }}</dd>
           {% endif %}
 
+          {% if package.timeout %}
+          <dt>Timeout:</dt>
+          <dd>{{ package.timeout }}</dd>
```

Comment:
Can you please do `{{ build.timeout|natural_time_delta() }}`, so the time is in a human-friendly format?

Reply:
Done.